samples/java-eventsourced-customer-registry/src/main/java/customer/domain/CustomerEntity.java

```diff
@@ -0,0 +1,123 @@
+/* This code was initialised by Akka Serverless tooling.
+ * As long as this file exists it will not be re-generated.
+ * You are free to make changes to this file.
+ */
+
+package customer.domain;
+
+import com.akkaserverless.javasdk.EntityId;
+import com.akkaserverless.javasdk.eventsourcedentity.CommandHandler;
+import com.akkaserverless.javasdk.eventsourcedentity.EventHandler;
+import com.akkaserverless.javasdk.eventsourcedentity.EventSourcedContext;
+import com.akkaserverless.javasdk.eventsourcedentity.EventSourcedEntity;
+import com.akkaserverless.javasdk.eventsourcedentity.EventSourcedEntityBase;
+import com.akkaserverless.javasdk.eventsourcedentity.EventSourcedEntityBase.Effect;
+import com.google.protobuf.Empty;
+import customer.api.CustomerApi;
+
+/** An event sourced entity. */
```

Comment:
Not necessarily for this PR but one thing that would be nice would be if we could include what protobuf it corresponds to here (like we have manually in some of the serverless framework test classes).